### PR TITLE
active_customer_db

### PR DESCRIPTION
**Pull request by pinkston4**
# Description
Wrote one test to see if we could return active customer from db. Wrote implementation code for test. Method called get_active in customer_db_interactor. Test now passes

# Related Ticket(s)
#6 #13  
## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation

[X] There is new documentation in this pull request that must be reviewed..

## Steps to Test
Outline the steps to test the code in this pull request.

1. fetch this branch
1. switch to this branch
1. navigate to tests folder
1. run command 
```
python test_customer.py
```

### CODE REVIEW

**Review by danirenae — APPROVED** [review]
Fetched and tested and it looks good! 👍

**Comment by samphillips1879** [comment]
test_can_return_active is sometimes failing, as depending on which order the tests are called there may not BE an active customer. Rewriting the test to ensure that an active customer is set should fix this

File: tests/test_customer.py [hunk 1]
```diff
@@ -83,6 +84,16 @@ def test_can_update_false(self):
 		for customer in new_customers_again:
 			self.assertEqual(0, customer[7])
 
+	def test_can_return_active(self):
+		"""
+		testing to see if you can return a customer with active set to true
+		"""
+		Customer_db().update_active_false()
```
Comment (samphillips1879) [inline tests/test_customer.py:91]: good job 💯

**Review by samphillips1879 — COMMENTED** [review]
Tests pass on my machine, code is commented, I say 👍

**Pull request merged**